### PR TITLE
Fix #1327 : Upgrade d-m-p

### LINKED ISSUE #1327: Upgrade d-m-p
### Description

Upgrade to the latest version of [docker-maven-plugin](https://github.com/fabric8io/docker-maven-plugin)

### PR DESCRIPTION
**Pull request by rohanKanojia**
Fixes #1327

### CODE REVIEW

File: parent/pom.xml [hunk 1]
```diff
@@ -351,6 +357,12 @@
         <version>2.8.7</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.codehaus.plexus</groupId>
+        <artifactId>plexus-interpolation</artifactId>
```
Comment (rhuss) [inline parent/pom.xml]: Do you know why this dependency is required ?

File: parent/pom.xml [hunk 2]
```diff
@@ -197,6 +197,12 @@
         <version>${project.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>com.google.code.findbugs</groupId>
+        <artifactId>jsr305</artifactId>
+        <version>3.0.2</version>
```
Comment (rhuss) [inline parent/pom.xml]: where is this used ?

File: parent/pom.xml (hunk 1 above)
Comment (rohanKanojia) [reply parent/pom.xml]: It's being used here: https://github.com/fabric8io/fabric8-maven-plugin/blob/7d7b13e3f675543d09e56eccc71f20abd42be0ae/core/src/main/java/io/fabric8/maven/core/service/openshift/OpenshiftBuildService.java#L426

File: parent/pom.xml (hunk 2 above)
Comment (rohanKanojia) [reply parent/pom.xml]: It is used for Nullable annotation here : https://github.com/fabric8io/fabric8-maven-plugin/blob/7d7b13e3f675543d09e56eccc71f20abd42be0ae/enricher/fabric8/src/main/java/io/fabric8/maven/enricher/fabric8/VertxHealthCheckEnricher.java#L66

**Pull request merged**